content/HostedOptions.tid

```diff
@@ -7,16 +7,16 @@ type: text/x-tiddlywiki
 
 There are several ways that you can have your TiddlyWiki data hosted online, making it easier to access and share your information.
 
-!~TiddlySpot
-[<img[//tiddlyspot.com/_ts/images/banner-logo.png][TiddlySpot|http://tiddlyspot.com]] ~TiddlySpot lets you put TiddlyWiki documents on the Web, and edit them directly in your browser. It doesn't yet offer full support for simultaneous editing by multiple users.
+!Tiddlyhost
+<html><div style="float:left;padding:0.2em 0.5em;"><a href="https://tiddlyhost.com/"><img style="height:2em;width:2em;" src="//raw.githubusercontent.com/simonbaird/tiddlyhost/main/rails/app/assets/images/logo-800-square.png"></a></div></html> [[Tiddlyhost|http://tiddlyhost.com]] lets you put TiddlyWiki documents on the Web, and edit them directly in your browser. Its focus is on single-user TiddlyWiki hence it does not support simultaneous editing by multiple users. (Click "Show more types" when [[creating|https://tiddlyhost.com/sites/new]] to find the "TiddlyWiki Classic" option.)
```

Comment:
Not sure how to size the image using TiddlyWiki markup, that's why I used raw html here.